code/modules/admin/smites/boneless.dm

```diff
@@ -1,6 +1,6 @@
 /// Gives the target critically bad wounds
 /datum/smite/boneless
-	name = ":B:oneless"
+	name = "I'm going to saw through your bones"
```

Comment:
```suggestion
	name = "Boneless"
```

Reply:
Boneless is also reddit

Reply:
A TF2 quote is peak zoomer and this fits with the naming scheme and the typepath

Reply:
Replaced.